ztp/resource-generator/README.md

```diff
@@ -0,0 +1,88 @@
+# README #
+
+This directory provides RAN site ArgoCD gitops pipeline container image definitions
+
+## What is this repository for? ##
+### Quick summary ###
+
+The idea is to accept RAN site manifests in a format defined by "siteconfig" CRD, convert them to custom resources accepted by the assisted service and ACM, and apply them to the cluster running Assisted installer operator and ACM hub.
+The realization is using [ArgoCD resource hooks](https://argoproj.github.io/argo-cd/user-guide/resource_hooks/)
+
+The pipeline would look like that:
+1. ArgoCD monitors site-plan repo for new site-plan data. The site plan data is expressed as a custom resource.
+2. Upon site plan change / push, ArgoCD synchronizes the site-plan data to the cluster. ArgoCD pre-sync and post-sync resource hooks are used to isolate the site configuration changes, convert them to changes in Assisted installer and ACM resources and apply to the cluster.
+The flow is shown below:
+
+<img src="assets/flow.png" width=800>
+
+## How do I get set up? ##
+### Prerequisites ###
+1. You need an Openshift cluster (version 4.8 or above)
+2. Red Hat GitOps operator installed
+3. Red Hat Advanced cluster Management for Kubernetes installed (version 2.3 or above)
+4. If you intend to build images yourself:
+   1. To push images you build, you need [quay.io](quay.io) credentials and belonging to [Red Hat employees organization in quay](https://quay.io/organization/redhat_emp1). But you can always fork this repo and push to your private account or another registry if desired  (Change the `REG ?= quay.io/redhat_emp1/ztp-site-generator` line in the [Makefile](Makefile) to the container registry of your choice)
```

Comment:
Also note that they need to update the pre and post hooks to point to their image.